config/environments/production.rb

```diff
@@ -41,11 +41,6 @@
   # Force all access to the app over SSL, use Strict-Transport-Security, and use secure cookies.
   config.force_ssl = true
 
-  # Log to STDOUT by default
-  config.logger = ActiveSupport::Logger.new($stdout)
-                                       .tap  { |logger| logger.formatter = Logger::Formatter.new }
-                                       .then { |logger| ActiveSupport::TaggedLogging.new(logger) }
-
```

Comment:
We didn't need to add this because this file already has https://github.com/sul-dlss/sdr-api/pull/731/files#diff-da60b4e96eff2b132991226d308949e23f4ef3aad45ad59edd09cbc32cc6251eR76-R80 which only lots to `STDOUT` if the `RAILS_LOG_TO_STDOUT` env var is turned on.